activiti-cloud-services-query/activiti-cloud-services-query-graphql/src/test/java/org/activiti/cloud/services/query/graphql/autoconfigure/EnableActivitiGraphQLQueryServiceTest.java

```diff
@@ -31,18 +31,18 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(
-    properties={"spring.activiti.cloud.services.query.graphql.enabled=false"},
+    properties={"spring.activiti.cloud.services.query.graphql.enabled=true"},
     webEnvironment = WebEnvironment.NONE
 )
 public class EnableActivitiGraphQLQueryServiceTest {
 
-    @Autowired
+    @Autowired(required = false)
```

Comment:
Are you sure that those beans are optional? The test will more likely fail anyway if they are not there.